tests/container_test.go

```diff
@@ -196,6 +196,7 @@ func TestContainerPut(t *testing.T) {
 		})
 		cNNS := c.CommitteeInvoker(nnsHash)
 		cNNS.Invoke(t, expected, "resolve", "mycnt.neofs", int64(nns.TXT))
+		c.Invoke(t, stackitem.NewBuffer([]byte("mycnt.neofs")), "alias", cnt.id[:])
```

Comment:
Could we add a test for missing alias to check the Null result?

Reply:
Sure, there you have it.